custom_model_runner/datarobot_drum/drum/args_parser.py

```diff
@@ -428,13 +428,23 @@ def _reg_args_monitoring(*parsers):
                 help="MLOps setting to use for connecting with the MLOps Agent (env: MONITOR_SETTINGS)",
             )
 
+    # TODO: restrict params to be used with unstructured target type only
+    @staticmethod
+    def _reg_args_params(*parsers):
+        for parser in parsers:
+            parser.add_argument(
+                ArgumentsOptions.PARAMS,
+                default=None,
+                help="Additional params",
+            )
+
     @staticmethod
     def _reg_arg_target_type(*parsers):
         target_types = [e.value for e in TargetType]
         for parser in parsers:
             parser.add_argument(
                 ArgumentsOptions.TARGET_TYPE,
-                required=True,
+                required=False,
```

Comment:
If its not required anymore, what means None then?

Reply:
Target type itself is required, but it can be obtained from either cli or model config file.
So default None is needed to correctly resolve final value